acme/src/test/groovy/io/micronaut/acme/AcmeCertRefresherMultiDomainsTaskSpec.groovy

```diff
@@ -52,13 +52,17 @@ class AcmeCertRefresherMultiDomainsTaskSpec extends AcmeBaseSpec {
                 try {
                     conn.connect()
                     Certificate[] certs = conn.getServerCertificates()
-                    certs.length == 1
+                    certs.length == 2
                     def cert = (X509Certificate) certs[0]
                     cert.getIssuerDN().getName().contains("Pebble Intermediate CA")
                     cert.getSubjectDN().getName().contains(EXPECTED_DOMAIN)
                     cert.getSubjectAlternativeNames().size() == 2
                     cert.getSubjectAlternativeNames().collect({d-> d.get(1)}).contains(EXPECTED_DOMAIN)
                     cert.getSubjectAlternativeNames().collect({d-> d.get(1)}).contains(EXPECTED_ACME_DOMAIN)
+
+                    def cert2 = (X509Certificate) certs[1]
+                    cert2.getIssuerDN().getName().contains("Pebble Root CA")
+                    cert2.getSubjectDN().getName().contains("Pebble Intermediate CA")
```

Comment:
In groovy, you can be more idiomatic by rewriting this:

`cert2.getSubjectDN().getName().contains("Pebble Intermediate CA")`

as: 

`cert2.subjectDN.name.contains("Pebble Intermediate CA")`